src/main/java/omero/gateway/facility/MetadataFacility.java

```diff
@@ -303,6 +307,35 @@ else if (!type.equals(PojoMapper.getModelType(obj.getClass())
         return Collections.emptyList();
     }
 
+    /**
+     * Get the plane infos.
+     * @param ctx The SecurityContext
+     * @param pix The PixelsData
+     * @return See above
+     * @throws DSOutOfServiceException
+     * @throws DSAccessException
+     */
+    public List<PlaneInfoData> getPlaneInfos(SecurityContext ctx, PixelsData pix) throws DSOutOfServiceException,
+            DSAccessException {
+        List<PlaneInfoData> result = new ArrayList<>();
+        try {
+            ParametersI p = new ParametersI();
+            p.addLong("id", pix.getId());
+            List<IObject> planeinfos = gateway.getQueryService(ctx)
```

Comment:
This should probably be declared in IMetadata.ice (blitz) and implemented in omero-server as we have done with other similar method calls.